frontend/package.json

```diff
@@ -7,11 +7,17 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
+    "format": "prettier --write . --plugin prettier-plugin-tailwindcss .",
     "test": "jest",
     "test:watch": "jest --watch"
   },
   "lint-staged": {
-    "*": "echo 'lint-staged frontend'"
+    "*.{js,ts,tsx,json}": [
+      "prettier --write --ignore-unknown"
+    ],
+    "*.{css,html}": [
+      "prettier --write --ignore-unknown --plugin prettier-plugin-tailwindcss"
+    ]
```

Comment:
**thought:** `tsx` components may have Tailwind classes

Reply:
Thanks for pointing that out! I've made an update that addresses tab width and formatting in TSX files.